package.json

```diff
@@ -25,7 +25,7 @@
     "test": "istanbul test -- _mocha",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls",
     "apidoc": "apidoc -o apidoc-out -i src/controllers/",
-    "integration": "integration all",
+    "integration": "CIRCLE_BRANCH=ilp_server_alpha integration all",
```

Comment:
Workaround CircleCI branch naming -- Circle was setting `pr/163` as the branch name.
cc @sentientwaffle